insalan/tournament/admin.py

```diff
@@ -856,3 +863,15 @@ def launch_swiss_matchs_action(self,request,queryset):
         self.message_user(request,_("Les matchs ont bien été lancés"))
 
 admin.site.register(SwissMatch, SwissMatchAdmin)
+
+
+class SeatingAdmin(admin.ModelAdmin):
```

Comment:
I don't thing it makes a lot of sense to edit seat with their coordinates, once we will have a proper admin page for it, this will not be needed anymore.